src/argilla/client/feedback/schemas.py

```diff
@@ -451,3 +499,263 @@ class FeedbackDatasetConfig(BaseModel):
 
     class Config:
         smart_union = True
+
+
+class RatingQuestionStrategy(Enum):
```

Comment:
These are not schemas as is, so I'd suggest to move this to another module under `feedback` or even to create a separate directory for `training` or `training_utils`

Reply:
I was unsure about the place for this but for me it seemed more generic, not specific to training. I will create a maybe a `unification` directory?

Reply:
Maybe a module named `unification.py` is enough, as we'll include there the different unification classes and that's it, right?

Reply:
Sure